docs/guide/docs/advanced/cluster.md

```diff
@@ -18,7 +18,7 @@ title: Clustering
 - Load Balancer with a public address
 - Botpress license registered with a public address
 
-## Enable cluster
+## Enable Redis Sentinel
```

Comment:
The example below is for a single node setup, isn't Sentinel for high availability & require an advanced setup?